server/data/cache/PrisonRegisterStore.ts

```diff
@@ -10,11 +10,11 @@ export default class PrisonRegisterStore extends RedisStore {
     super(redisClient)
   }
 
-  public async setActivePrisons(activePrisons: Array<Prison>, durationDays = 1): Promise<void> {
-    return this.setRedisAsync(ACTIVE_PRISONS, JSON.stringify(activePrisons), 'EX', durationDays * 24 * 60 * 60)
+  public setActivePrisons(activePrisons: Array<PrisonDto>, durationDays = 1) {
```

Comment:
Also changed the signature of `setActivePrisons` to not return the promise - we are not dependant on its return, and we dont wait for its return. It makes the calling code a little cleaner because the linter doesn't issue a warning that we are not doing anything with the return value 👍